Solutions/Claroty xDome/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "claroty",
+	"offerId": "microsoft-sentinel-solution-xdome",
+	"firstPublishDate": "2024-02-01",
+	"providers": ["Claroty"],
+	"categories": {
+		"domains" : ["Security - Threat Intelligence ", "Security - Vulnerability Management"],
```

Comment:
Hi @roysagi, Please correct domain category

Reply:
done

Reply:
@v-prasadboke - Hi,
It seems I am failing on the domains validation even though the domains I have listed appear [here](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/sentinel/sentinel-solutions.md?msclkid=9a240b52b11411ec99ae6736bd089c4a#categories-for-microsoft-sentinel-out-of-the-box-content-and-solutions) and I see them used in other solutions such as here:
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/QualysVM/SolutionMetadata.json
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/CTM360/SolutionMetadata.json